example/components/Content/NonLoggedInView/NonLoggedInView.tsx

```diff
@@ -6,14 +6,15 @@ const NonLoggedInView = () => {
   const { openConnectModal } = useConnectModal();
 
   return (
-    <Flex direction={'column'} alignItems="flex-start" maxW="424px" gap={5}>
-      <Text fontSize="3xl">Token Gate Demo Website</Text>
-      <Text fontSize="xl">
+    <Flex direction={'column'} alignItems="flex-start" maxW="454px" gap={5} px="40px">
+      <Text fontSize="3xl">Token.Gate Demo Website</Text>
```

Comment:
I prefer keeping the name `Token Gate`

Reply:
I would actually change it to `Collab.Land Token Gate`, but if I have to vouch for one of those two I would pick `Token Gate`